<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowControlWidget</class>
 <widget class="QWidget" name="ShowControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="showLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="title">
      <string>Show Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_18">
      <item row="1" column="0">
       <widget class="QCheckBox" name="lockMaestroCheckBox">
        <property name="toolTip">
         <string>If enabled, actions will no longer update the Maestro but will still be listed in the Event History</string>
        </property>
        <property name="text">
         <string>Lock Maestro</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableCheckBox">
        <property name="toolTip">
         <string>Enables support for queuing actions</string>
        </property>
        <property name="text">
         <string>Enable Show</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="resyncMaestroButton">
        <property name="toolTip">
         <string>Reinitializes the last run time of all timers</string>
        </property>
        <property name="text">
         <string>Resync Timers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="advancedSettingsGroupBox">
     <property name="title">
      <string>Advanced Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="1" column="1">
       <widget class="QCheckBox" name="loopCheckBox">
        <property name="toolTip">
         <string>Repeats the Show after the last Event (Relative timing only)</string>
        </property>
        <property name="text">
         <string>Loop Events</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="timingModeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Timing Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="timingModeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Sets the timing method. Absolute tracks from the current time, while Relative tracks from the last Event time</string>
        </property>
        <item>
         <property name="text">
          <string>Absolute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relative</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="eventsGroupBox">
     <property name="title">
      <string>Events</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_17">
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="eventControlLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="absoluteTimeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Absolute Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="absoluteTimeLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The Maestro's total runtime</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="relativeTimeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Relative Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="relativeTimeLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The amount of time since the last event</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eventTimeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Event Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="eventTimeSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The time that the Event will run</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="eventEditGridLayout">
          <item row="1" column="2">
           <widget class="QPushButton" name="addEventButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Adds the selected Event(s) to the Event List</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/images/images.qrc">
              <normaloff>:/icon_next.png</normaloff>:/icon_next.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="removeEventButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Removes the selected Event(s) from the Event List</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/images/images.qrc">
              <normaloff>:/icon_back.png</normaloff>:/icon_back.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="moveEventUpButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/images/images.qrc">
              <normaloff>:/icon_up.png</normaloff>:/icon_up.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="moveEventDownButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/images/images.qrc">
              <normaloff>:/icon_down.png</normaloff>:/icon_down.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="playPauseButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/images/images.qrc">
              <normaloff>:/icon_pause.png</normaloff>
              <normalon>:/icon_play.png</normalon>:/icon_pause.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="eventHistoryLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="eventHistoryWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The most recent actions performed in the Animation Editor</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QListWidget" name="eventQueueWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Lists all of the Show's events</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="eventListLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Queue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
